Mathlib/RingTheory/Valuation/Discrete/Basic.lean

```diff
@@ -3,52 +3,65 @@ Copyright (c) 2025 María Inés de Frutos-Fernández, Filippo A. E. Nuccio. All
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: María Inés de Frutos-Fernández, Filippo A. E. Nuccio
 -/
-import Mathlib.Algebra.GroupWithZero.Int
+import Mathlib.Algebra.Order.Group.Cyclic
+import Mathlib.Algebra.Order.Group.Units
 import Mathlib.RingTheory.Valuation.Basic
 
 /-!
 # Discrete Valuations
 
-A valuation `v : A → ℤₘ₀` on a ring `A` is said to be a (normalized) discrete valuation if
-`ofAdd (-1 : ℤ)` belongs to the image of `v`. Note that valuations in Mathlib are multiplicative;
-if `a : A → ℤ ∪ {infty}` is the additive valuation associated to `v`, this is equivalent to asking
-that `1 : ℤ` belongs to the image of `a`.
+Given a linearly ordered commutative group with zero `Γ` such that `Γˣ` is nontrivial cyclic, a
+valuation `v : A → Γ` on a ring `A` is *discrete*, if `genLTOne Γˣ` belongs to the image. When
+`Γ := ℤₘ₀`, the latter is equivalent to asking that `ofAdd (-1 : ℤ)` belongs to the image, in turn
```

Comment:
You might also want to explain what `ℤₘ₀` is, I am imagining random PhD students reading this file in July who might not know their way around our funny notations.

Reply:
Good point, done.